packages/web-components/package.json

```diff
@@ -38,7 +38,7 @@
   "devDependencies": {
     "@axe-core/puppeteer": "^4.10.0",
     "@babel/core": "^7.12.13",
-    "@department-of-veterans-affairs/css-library": "^0.21.2",
+    "@department-of-veterans-affairs/css-library": "^0.22.1",
```

Comment:
This this need to be published still? NPM is still showing `0.22.0`: https://www.npmjs.com/package/@department-of-veterans-affairs/css-library